test/test.py

```diff
@@ -49,9 +49,7 @@
 True
 >>> run({'cycle': 2, 'pre': 3, 'unit': 1, 'ratio': [1, 0, 3, 0]})
 Preparing . . .
-1
-2
-3
+. . . 
```

Comment:
@sadrasabouri Thanks for your effort.
But I think it would be better if we have only one `. . .`, only after `Preparing`

Reply:
OK.
Thank you for review, this change considered in [54b0647](https://github.com/sepandhaghighi/nafas/pull/17/commits/54b06479094d4c2b8139fe11bab661127148b6c5).